doc/api/process.markdown

```diff
@@ -903,6 +913,38 @@ a diff reading, useful for benchmarks and measuring intervals:
     }, 1000);
 
 
+## process.send(message[, sendHandle])
```

Comment:
as of 607aa3ac82cb8c07487b474e548aa205e8ee3ca5 this should be `process.send(message[, sendHandle][, callback])` I believe